### ISSUE TITLE
Windows Python 3.8 Tornado asyncio Event loop Error

### ISSUE THREAD

**Issue by daniel-butler** [jiffyclub/snakeviz#138]
Based on [this issue in tornado](https://github.com/tornadoweb/tornado/issues/2608). Snakeviz needs to set up a selector event loop for tornado to work properly in python 3.8.

This is the error message.
```powershell
Traceback (most recent call last):
  File "c:\users\docadmin\appdata\local\programs\python\python38-32\Lib\runpy.py", line 192, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "c:\users\docadmin\appdata\local\programs\python\python38-32\Lib\runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "C:\ProgramData\CoreCustom\electronic-payments\venv3-8-0\Scripts\snakeviz.exe\__main__.py", line 7, in <module>
  File "c:\programdata\corecustom\electronic-payments\venv3-8-0\lib\site-packages\snakeviz\cli.py", line 123, in main
    app.listen(p, address=hostname)
  File "c:\programdata\corecustom\electronic-payments\venv3-8-0\lib\site-packages\tornado\web.py", line 2112, in listen
    server.listen(port, address)
  File "c:\programdata\corecustom\electronic-payments\venv3-8-0\lib\site-packages\tornado\tcpserver.py", line 152, in li
sten
    self.add_sockets(sockets)
  File "c:\programdata\corecustom\electronic-payments\venv3-8-0\lib\site-packages\tornado\tcpserver.py", line 165, in ad
d_sockets
    self._handlers[sock.fileno()] = add_accept_handler(
  File "c:\programdata\corecustom\electronic-payments\venv3-8-0\lib\site-packages\tornado\netutil.py", line 279, in add_
accept_handler
    io_loop.add_handler(sock, accept_handler, IOLoop.READ)
  File "c:\programdata\corecustom\electronic-payments\venv3-8-0\lib\site-packages\tornado\platform\asyncio.py", line 99,
 in add_handler
    self.asyncio_loop.add_reader(fd, self._handle_events, fd, IOLoop.READ)
  File "c:\users\docadmin\appdata\local\programs\python\python38-32\Lib\asyncio\events.py", line 501, in add_reader
    raise NotImplementedError
NotImplementedError
```

**Comment by Alexander-Serov** [comment]
Hello!

Is there any news on this issue? A workaround?

**Comment by jiffyclub** [comment]
Fixed via #139 and released in version 2.1.0. Thanks for reporting and submitting a fix for this!
